media-gfx/gimp/gimp-2.9.8-r1.ebuild

```diff
@@ -94,6 +96,10 @@ PATCHES=(
 )
 
 src_prepare() {
+	# Disable system CFLAGS suppressing on SSE{2,4.1} support tests by addition of {SSE2,SSE4_1}_EXTRA_CFLAGS: bug #702554
+	sed -i -e 's:\$intrinsics_save_CFLAGS \$SSE2_EXTRA_CFLAGS:\$SSE2_EXTRA_CFLAGS \$intrinsics_save_CFLAGS:' configure.ac ||  die
+	sed -i -e 's:\$intrinsics_save_CFLAGS \$SSE4_1_EXTRA_CFLAGS:\$SSE4_1_EXTRA_CFLAGS \$intrinsics_save_CFLAGS:' configure.ac ||  die
```

Comment:
Sorry for bouncing this off, but we really dislike calling sed multiple times when editing one file. You can make multiple different sed edits per file, with one call.

Reply:
If those are env variables, I think you're better using `{ }` around them as well.

Reply:
I combined this sed but still leave separate calls for different issues.

The swaped variables are internal temporary configure.ac variables inside assignments. One is used to store initial value of CFLAGS and others are used to add additional GCC options.